commit d867a5a33c71ff3a03b890b46f20c4fd4d1cab35 (refs/changes/93/4393/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2018-06-20T11:00:03-07:00 (1 year, 4 months ago)
    
    TRITON-530 nightly-1 vmapi fabric provision test is failing on "should have found one NAT vm"

diff --git a/test/vms.full.test.js b/test/vms.full.test.js
index 01e11b3..243fb6e 100644
--- a/test/vms.full.test.js
+++ b/test/vms.full.test.js
@@ -2798,8 +2798,10 @@ exports.destroy_test_vms_final = function (t) {
 exports.find_fabric_network = function (t) {
     assert.arrayOfObject(NETWORKS, 'NETWORKS');
 
+    // Find a fabric network that is owned by CUSTOMER.
     fabricNetwork = NETWORKS.find(function _findFabricNetwork(n) {
-        return n.fabric === true;
+        return n.fabric === true && Array.isArray(n.owner_uuids) &&
+            n.owner_uuids.indexOf(CUSTOMER) >= 0;
     });
 
     t.ok(fabricNetwork, 'found a fabric network');
